src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj

```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```

Comment:
This seems a little unnatural. It seems like this should be changed to an sdk style project? @sbomer any reason you know of that would inhibit this?

Reply:
This is a `*.pkgproj` so I don't think there is any SDK support. What this really should be is a blank/custom style. Remove the `ToolsVersion` and `xmlns` attributes.